rfcs/0088-nixpkgs-breaking-change-policy.md

```diff
@@ -24,7 +24,7 @@ The direct intended result of this RFC is:
 
 - Maintainers of dependencies have a clear framework for handling changes that
   break dependants.
-- Maintainers of dependants have a clear framework for how dependency breacking
+- Maintainers of dependants have a clear framework for how dependency breaking
```

Comment:
While we're here...

```suggestion
- Maintainers of dependents have a clear framework for how dependency breaking
```

Reply:
Oh I assumed it was correct, but that's misspelled everywhere then?

Reply:
Apparently the are both correct. So I guess we should probably stay consistent.

https://www.merriam-webster.com/words-at-play/spelling-variants-dependent-vs-dependant